k8s/fluentd/fluentd.t

```diff
@@ -0,0 +1,80 @@
+# Load values
+{{ load_module('legion.template_plugins.file_change_monitor', filepath='/opt/config/values.yaml', is_yaml=True, var_name='cfg') }}
+
+# Load secrets
+{{ load_module('legion.template_plugins.file_change_monitor', filepath='/opt/secrets/aws.yaml', is_yaml=True, var_name='aws_secrets') }}
+
+# Receive
+<source>
+  @type http
+  port 80
+</source>
+
+# Processing rules
+# docs: https://github.com/fluent/fluent-plugin-s3
+
+{% for tag in cfg.specific %}
+# Section for tag {{ tag }}
+<match *>
+  @type s3
+  # Connection
+  aws_key_id {{ aws_secrets.aws_key_id }}
+  aws_sec_key {{ aws_secrets.aws_sec_key }}
+
+  s3_bucket {{ cfg.common.bucket }}
+  s3_region {{ cfg.common.region }}
+
+  # Storing
+  path {{ cfg.specific[tag].path }}
+  time_slice_format {{ cfg.specific[tag].time_slice_format }}
+  time_slice_wait {{ cfg.specific[tag].time_slice_wait }}
+  s3_object_key_format {{ cfg.specific[tag].s3_object_key_format }}
+  utc
+  store_as {{ cfg.specific[tag].store_as }}
+
+  # Buffering
+  <buffer tag,time>
+    @type file
+    path {{ cfg.specific[tag].buffer_path }}
+    timekey {{ cfg.specific[tag].timekey }}
+    timekey_wait {{ cfg.specific[tag].timekey_wait }}
+    timekey_use_utc true
+  </buffer>
+  <format>
+    @type {{ cfg.specific[tag].format_type }}
+  </format>
+</match>
+{% endfor %}
+
+{% if cfg is defined %}
+# Default
+<match *>
+  @type s3
+  # Connection
+  aws_key_id {{ aws_secrets.aws_key_id }}
```

Comment:
Can this be configured to use kube2iam role?

Reply:
@alexander-semenets is kube2iam ready to use?

Reply:
@akharlamov kube2iam-branch is not ready yet